src/components/Nav/Nav.js

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import { BrowserRouter as Router, Switch, Route, Link } from "react-router-dom";
+import "../../../node_modules/bootstrap/dist/css/bootstrap.min.css";
+import Image from "../../static/images/Logo-Blue.png";
+import "./Nav.css";
+
+
+export default function Navbar() {
+  return (
+    <div className="navbar navbar-expand-lg bg-dark fixed-top">
+      <Link to={"/"} className="brand">
+        <img alt="logo" src={Image} height="60" />
```

Comment:
Should the height not be specified inside a style property? like:
`style={{height: "60px"}}`

Reply:
this is not styling its an attribute

Reply:
it makes no difference but I've changed it to styling